### ISSUE TITLE
Fix TypeError with postcss-import plugin in at-rule-name-space-after

### ISSUE THREAD

**Issue by hail2u** [stylelint/stylelint#4801]
> Clearly describe the bug

If a CSS file contains `@import` with media queries (see below CSS example), using stylelint as a PostCSS plugin with postcss-import plugin will throw an TypeError.

> Which rule, if any, is the bug related to?

`at-rule-name-space-after`

> What code is needed to reproduce the bug?

foo.css:

```css
@import "bar.css" all;
```

bar.css:

```css
/* bar.css */
```

index.js:

```javascript
import fs from "fs/promises";
import pcImport from "postcss-import";
import postcss from "postcss";
import stylelint from "stylelint";

(async () => {
	const css = await fs.readFile("foo.css", "utf8");
	await postcss()
		.use(pcImport)
		.use(stylelint({
			"fix": true
		}))
		.process(css, {
			"from": "foo.css"
		});
})().catch((e) => {
	console.trace(e);
	process.exitCode = 1;
});
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
	"rules": {
		"at-rule-name-space-after": "always"
	}
}
```

> Which version of stylelint are you using?

`13.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

As PostCSS plugin with `postcss-import`.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

Script (`index.js`) exits without throwing an error.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
$ node index.js
Trace: TypeError: Cannot read property 'replace' of undefined
    at C:\sandbox\bar.css:1:1
    at Object.atRuleNameSpaceChecker.fix (C:\sandbox\node_modules\stylelint\lib\rules\at-rule-name-space-after\index.js:36:53)
    at Object.err (C:\sandbox\node_modules\stylelint\lib\rules\atRuleNameSpaceChecker.js:27:14)
    at expectAfter (C:\sandbox\node_modules\stylelint\lib\utils\whitespaceChecker.js:318:14)
    at Object.after [as locationChecker] (C:\sandbox\node_modules\stylelint\lib\utils\whitespaceChecker.js:156:5)
    at checkColon (C:\sandbox\node_modules\stylelint\lib\rules\atRuleNameSpaceChecker.js:22:11)
    at C:\sandbox\node_modules\stylelint\lib\rules\atRuleNameSpaceChecker.js:14:3
    at C:\sandbox\node_modules\postcss\lib\container.js:295:18
    at C:\sandbox\node_modules\postcss\lib\container.js:135:18
    at Root.each (C:\sandbox\node_modules\postcss\lib\container.js:101:16)
    at Root.walk (C:\sandbox\node_modules\postcss\lib\container.js:131:17) {
  postcssNode: AtRule {
    raws: {},
    name: 'media',
    params: 'all',
    source: { input: [Input], start: [Object] },
    type: 'atrule',
    parent: Root {
      raws: [Object],
      type: 'root',
      nodes: [Array],
      source: [Object],
      lastEach: 3,
      indexes: [Object]
    },
    nodes: [ [Comment] ],
    lastEach: 1,
    indexes: {}
  }
}
    at file:///C:/sandbox/index.js:17:10
```

**Comment by jeddy3** [comment]
@hail2u Thanks for the very detailed report and for using the template.

I'll label this as a bug. However, it's going against the recommended usage. stylelint is meant to be used on your source files and not on your compiled CSS. I suspect postcss-import is doing something to the AST that is causing stylelint to trip up.

I suggest using stylelint as a standalone tool first because you get better reporting among other things. If you really want to use it as a PostCSS plugin, then either run it as a separate task or use the `plugins` property of `postcss-import` as showed [in the usage examples](https://stylelint.io/user-guide/usage/postcss-plugin#usage-examples).

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and fix this bug.
